src/pedalintandem.py

```diff
@@ -0,0 +1,114 @@
+import json
+from datetime import datetime
+from curl_cffi import requests
+from common.tz import IST
+from bs4 import BeautifulSoup
+
+BASE_URL = "https://www.pedalintandem.com"
+
+def _date(date_str):
+    return datetime.fromisoformat(date_str).astimezone(IST).isoformat()
+
+
+def fetch_events_links(session):
+	res = session.get(f"{BASE_URL}/experiences", impersonate="chrome")
+	soup = BeautifulSoup(res.text, 'html.parser')
+	event_divs = soup.find_all('div', class_ = 'single-experience')
+
+	# Fetch events from other pages.
+	load_more = soup.find('div', class_ = 'products-loadmore')
+	while load_more != None:
+		url = load_more.find('a').get('href')
+		
+		new_page = session.get(f"{BASE_URL}{url}")
+		soup = BeautifulSoup(new_page.text, 'html.parser')
+		
+		new_events = soup.find_all('div', class_ = 'single-experience')
+		event_divs += new_events
+
+		load_more = soup.find('div', class_ = 'products-loadmore')
+
+	event_links = []
+	for event_div in event_divs:
+		url = event_div.find('a').get('href')
+		event_links.append(url)
+
+	return event_links
+
+def fetch_events(event_links, session):
+	events = []
+	for event_link in event_links:
+		event = []
+
+		event_page = session.get(f"{BASE_URL}{event_link}", impersonate='chrome')
+		event = BeautifulSoup(event_page.text, 'html.parser')
+		date_selector = event.find('div', 'product-variations-varieties').find('select')
+
+		# Fetch data of the events which would be happening in future otherwise skip
+		if "disabled" in date_selector.attrs:
+			continue
+
+		events.append(event)
+
+	return events
+
+def make_event(event):
```

Comment:
Add some comments as well please. 
```suggestion
def make_event(soup):
```

Reply:
Will change and add comments.